src/main/flight/imu.h

```diff
@@ -59,7 +59,7 @@ typedef struct imuConfig_s {
     uint16_t imu_dcm_ki;          // DCM filter integral gain ( x 10000)
     uint8_t small_angle;
     uint8_t imu_process_denom;
-    uint16_t mag_declination;     // Magnetic declination in degrees * 10
+    int8_t mag_declination;       // Magnetic declination in degrees
```

Comment:
Another breaking change

Reply:
@ledvinap reverted

Reply:
@ledvinap is there anything left to be done here ?